app/models/groupe_instructeur_service.rb

```diff
@@ -0,0 +1,21 @@
+class GroupeInstructeurService < ApplicationRecord
```

Comment:
Du coup

```suggestion
class ContactInformation < ApplicationRecord
```

Je sais que c'est relou, mais j'ai l'impression que ça éviterait des confusions lorsqu'on parlera de service